### ISSUE TITLE
[BUG] Header items with No Sub Items should be links

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#158]
**Description**
Using a button instead of link to navigates users voids the inbuilt functionality that 'a' tags have. Such as telling assistive tech user the href to inform them of the destination, assistive tech tools provide a list of links, headings and landmarks that user can use to short cuts to navigate too.  

**Expected behaviour**
The 'button' element should be used for any interaction that performs an action on the current page. The 'a' element should be used for any interaction that navigates to another view.

**Screenshots**
![image](https://github.com/UKHO/admiralty-design-system/assets/97240814/e6ce9ab2-4a27-47e5-bdb2-c0e08299a5c2)
